nuxt.config.js

```diff
@@ -96,9 +96,33 @@ export default defineNuxtConfig({
     modules: [
         '@nuxtjs/i18n',
         '@pinia/nuxt',
-        'nuxt-viewport'
+        'nuxt-viewport',
     ],
 
+    viewport: {
+        breakpoints: {
+            desktop: 1024,
+            desktopMedium: 1280,
+            desktopWide: 1600,
+
+            mobile: 320,
+            mobileMedium: 375,
+            mobileWide: 425,
+
+            tablet: 768,
```

Comment:
Instead of a bunch of device breakpoints, what are your thoughts on using "portrait" vs "landscape" orientation only? 

The size doesn't really matter except if it's really small, then you force to portrait mode.

Reply:
@ermish These are the default values, but I think we need to differentiate at least between tablet and desktop, because tablet is experiencing bugs with the top nav and the map is barely visible.

Either way, I'd like to at least get the mobile site working first, and then nitpick on the custom breakpoints that are being used. Currently, the site is still completely broken for mobile without this PR and I'd like to get it to a usable state.

Reply:
Yeah we'll need to make it more responsive. Ok we can figure out later.